# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296481018 18000
#      Mon Jan 31 08:36:58 2011 -0500
# Branch release-3-4-x
# Node ID b10ea6efdc58565109152b435ac7b9b1a7697223
# Parent  9fa77a1ead19b8b4275961feb1e8e842306f54da
version is now 3.3.91

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,14 @@
+2011-01-31  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac (AC_INIT): Version is now 3.3.91.
+	(OCTAVE_API_VERSION_NUMBER): Now 45.
+	(OCTAVE_RELEASE_DATE): Now 2011-01-31.
+
 2010-01-30  Rik  <octave@nomad.inbox5.com>
 
 	* NEWS: Deprecate glpkmex function.
 	
 2011-01-30  John W. Eaton  <jwe@octave.org>
 
 	* acinclude.m4 (OCTAVE_PROG_GHOSTSCRIPT): Update warning message.
 
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -22,23 +22,23 @@ dnl Copyright (C) 1993-2011 John W. Eato
 
 ### Preserve CFLAGS and CXXFLAGS from the environment before doing
 ### anything else because we don't know which macros might call
 ### AC_PROG_CC or AC_PROG_CXX.
 
 EXTERN_CFLAGS="$CFLAGS"
 EXTERN_CXXFLAGS="$CXXFLAGS"
 
-AC_INIT([GNU Octave], [3.3.90+], [bug@octave.org], [octave])
+AC_INIT([GNU Octave], [3.3.91], [bug@octave.org], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
-OCTAVE_API_VERSION_NUMBER="44"
+OCTAVE_API_VERSION_NUMBER="45"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
-OCTAVE_RELEASE_DATE="2011-01-22"
+OCTAVE_RELEASE_DATE="2011-01-31"
 OCTAVE_COPYRIGHT="Copyright (C) 2011 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
 AC_SUBST(OCTAVE_API_VERSION_NUMBER)
 AC_SUBST(OCTAVE_API_VERSION)
 AC_SUBST(OCTAVE_RELEASE_DATE)
 AC_SUBST(OCTAVE_COPYRIGHT)
 
 AC_REVISION($Revision: 1.603 $)
